Tools/nib2cib/NSBox.j

```diff
@@ -71,9 +71,8 @@
         }
 
         _title         = [[aCoder decodeObjectForKey:@"NSTitleCell"] objectValue] || @"";
-        _titlePosition = [aCoder decodeObjectForKey:@"NSTitlePosition"];
 
-        if (_titlePosition === undefined)
+        if (_titlePosition === undefined && _boxType !== CPBoxSeparator)
```

Comment:
I don't think that line 74 should be removed. What we need is just the extra test ```_boxType !== CPBoxSeparator```. @enquora can you please put the line back again and I think we should be ok with this.